textproc/dos2unix/Portfile

```diff
@@ -3,12 +3,11 @@
 PortSystem          1.0
 
 name                dos2unix
-version             7.4.3
+version             7.5.1
 revision            0
 categories          textproc
-platforms           darwin
 license             BSD
-maintainers         nomaintainer
+maintainers         @therealketo openmaintainer
```

Comment:
please include an obfuscated e-mail address as outlined in the guide.

Reply:
This is never directly stated, but I'll add one either way.